tokens/src/core/global/display.json

```diff
@@ -2,19 +2,19 @@
   "typography": {
     "display": {
       "line-height": {
-        "base": { "value": "1", "type": "dimension", "source": "$display-line-height" },
-        "mobile": { "value": "3.5rem", "type": "dimension", "source": "$display-mobile-line-height" }
+        "base": { "value": "1", "type": "dimension", "source": "$display-line-height", "description": "Line height basic value." },
+        "mobile": { "value": "3.5rem", "type": "dimension", "source": "$display-mobile-line-height", "description": "Line height for mobile devices." }
```

Comment:
Maybe something like "Standard line height" and "Mobile line height" instead here? To reiterate, don't take these comments as prescriptive, I'm just trying to spark conversation around what our description conventions should be.

Reply:
I think you variant is more human-friendly, so I would use it instead 🙂 Also, I think we definitely should avoid using word `value` in the description, it just doesn't contain any useful information for the reader.

Reply:
Changed to "Standard line height" and "Mobile line height"